tasks/docker_tasks.py

```diff
@@ -53,9 +53,9 @@ def dockerize_test(ctx, binary, skip_cleanup=False):
 
     with open(f"{temp_folder}/Dockerfile", 'w') as stream:
         stream.write(
-            """FROM ubuntu:20.04
-COPY --from=docker/compose-bin:v2.26.1 /docker-compose /usr/bin/compose
-COPY --from=docker:26.1-cli /usr/local/bin/docker /usr/bin/docker
+            """FROM public.ecr.aws/docker/library/ubuntu:20.04
+COPY --from=public.ecr.aws/docker/library/docker:26.1-cli /usr/local/bin/docker /usr/bin/docker
+RUN wget https://github.com/docker/compose/releases/download/v2.26.1/docker-compose-linux-x86_64 -O /usr/bin/compose
```

Comment:
Can we check the sha of the downloaded binary?
Something similar to that one: https://github.com/DataDog/datadog-agent/blob/main/.gitlab/deps_build/deps_build.yml#L15-L19

Reply:
Good point, fixed in [b075a9e](https://github.com/DataDog/datadog-agent/pull/30699/commits/b075a9e05bf0d6a60474c6ea054cf396af0d3b4f)